### ISSUE TITLE
Enhancement: pdf text and background color settings

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#446]
```
It will be great to be able to change the default black/white color scheme for 
"standard" pdf text because it is sometimes too contrasty either in day or 
night modes.

For example, coolreader have excellent support for this case.
One can change day/night colors of text and background and even use pleasant 
bitmap for the background.

Some pdf readers on market allows user to select one of predefined color 
schemes, but there is too common cases to be really useful for anyone.

Thank you in advance.
```

Original issue reported on code.google.com by `Kadav...@gmail.com` on 5 Feb 2013 at 10:32

**Comment by GoogleCodeExporter** [comment]
Original comment by `Andrei.K...@gmail.com` on 12 Feb 2013 at 12:30
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
There are situations when book after applying "fit to height" gets black areas 
on both sides. Would be great to have possibility to change background color to 
white.

Hope this makes sense.

Thanks in advance.


```

Original comment by `stepan.m...@gmail.com` on 29 Jun 2013 at 6:42

**Comment by GoogleCodeExporter** [comment]
```
Implemented in 2.0 
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 25 Dec 2013 at 10:32
- Changed state: **Fixed**
